Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -166,28 +166,16 @@ private string DeclareErrorExchangeWithQueue(IModel model, MessageReceivedInfo r
         MessageReceivedInfo receivedInfo, MessageProperties properties, byte[] body, Exception exception
     )
     {
-        var messageAsString = errorMessageSerializer.Serialize(body);
-        var patchedProperties = (MessageProperties)properties.Clone();
-        patchedProperties.Headers = PatchHeaders(patchedProperties.Headers);
-
         var error = new Error(
             receivedInfo.RoutingKey,
             receivedInfo.Exchange,
             receivedInfo.Queue,
             exception.ToString(),
-            messageAsString,
+            errorMessageSerializer.Serialize(body),
             DateTime.UtcNow,
-            patchedProperties
+            properties
         );
-
         return serializer.MessageToBytes(typeof(Error), error);
     }
 
-    private static IDictionary<string, object?> PatchHeaders(IDictionary<string, object?> headers)
```

Comment:
With out it, byte headers will not be sent correctly via hosepipe. I am going to roll this PR back.